community-concepts/productaanvraag/README-denhaag.md

```diff
@@ -0,0 +1,12 @@
+## ProductAanvraag
+
+Some properties are not reflected in the JSON-schema and are shown below.
+
+- Stereotype: IMBOR-Objecttype
```

Comment:
@joeribekker I'm not sure what this means and whether it applies here

Reply:
Ah, you can remove this line. This only holds for Objecttypes taken from the Gemeentelijk Gegevensmodel by Delft.